crates/uv/tests/common/mod.rs

```diff
@@ -295,6 +301,7 @@ impl TestContext {
             .arg(self.cache_dir.path())
             .env("VIRTUAL_ENV", self.venv.as_os_str())
             .env("UV_NO_WRAP", "1")
+            .env("UV_TEST_PYTHON_PATH", "/dev/null")
```

Comment:
Should we use empty string? Would it be more portable?

Reply:
I was worried we might treat an empty string as an empty variable but I guess we don't after looking at the source. Since this passes on Windows CI I think I'd prefer to leave it as-is because the intent is clearer? 🤷‍♀️